README.md

```diff
@@ -55,24 +55,9 @@ trainer(output_directory: ENV["CIRCLE_TEST_REPORTS"])
 
 For more information, check out the [fastlane plugin docs](fastlane-plugin-trainer#readme).
 
-## Without [fastlane](https://fastlane.tools)
-
-### Installation
```

Comment:
Why this part not any more?

Reply:
AFAICT, there is no reason to have a `Without` section because there is no way to use this gem without fastlane.

It could be argued that you don't need to use this gem as fastlane plugin - the instructions that I delete would work.  However, as they were written they were confusing - they implied that the gem did not need fastlane to be installed.

I can roll back that doc change if you want.

Reply:
Ah, interesting. I think that's more for when you don't use fastlane with a Fastfile, no?

Reply:
> I think that's more for when you don't use fastlane with a Fastfile

The implication, to me, was that fastlane was not required.

Reply:
Removed doc commit.